app/views/repositories/show.html.erb

```diff
@@ -34,15 +34,13 @@
       <% end %>
     </div>
     <div class="repo-datatables-buttons">
-      <div class="new-repository-button">
```

Comment:
We need class `new-repository-button` for adding `add_share_inventory_promo_modal` in billing addon.

Reply:
Fix in addon?